lib/omniauth-dsds/spec/sign_in_helper.rb

```diff
@@ -27,10 +27,18 @@ def mock_profile(options: {}, authentication_site_url: "http://app.example.com")
                 "postcode" => options.fetch(:postcode) { "" }
               },
               "PIN" => options.fetch(:pin) { "1234" },
-              "organisation_uids" => options.fetch(:organisation_uids) { ["12345678-bcde-1234-abcd-1234567890","12345678-cdef-1234-abcd-1234567890"] },
+              "organisations" => options.fetch(:organisations) do
+                [
+                    {
+                        "uid" => "12345678-bcde-1234-abcd-1234567890",
+                        "name" => "SOME ORGANISATION",
+                        "type" => %w(custody_suite law_firm).sample,
```

Comment:
Could we just pick a type to return? I have a feeling having a random type returned might cause pain.